src/plot/util.typ

```diff
@@ -303,6 +303,9 @@
       axis.min -= 1; axis.max += 1
     }
 
+    axis.mode = get-axis-option(name, "mode", "lin")
+    axis.base = get-axis-option(name, "base", 10)
```

Comment:
I imagine this could be useful for other things, like the ticks PR you have at the moment #18

Reply:
If we merge #18, you could pick the default formatter based on the axis options :+1:.
The formatter itself does not know about the axis.